pkg/pillar/cmd/zedrouter/ipsec.go

```diff
@@ -445,9 +444,9 @@ func ipRouteCreate(vpnConfig types.VpnConfig) error {
 	if vpnConfig.IsClient {
 		clientConfig := vpnConfig.ClientConfigList[0]
 		tunnelConfig := clientConfig.TunnelConfig
-		cmd := exec.Command("ip", "route", "add", gatewayConfig.SubnetBlock,
-			"dev", tunnelConfig.Name, "metric", tunnelConfig.Metric)
-		if _, err := cmd.Output(); err != nil {
+		if _, err := base.ExecCommand(log, false, false,
+			"ip", "route", "add", gatewayConfig.SubnetBlock, "dev", tunnelConfig.Name,
+			"metric", tunnelConfig.Metric); err != nil {
```

Comment:
Better to fix this to use netlink to add routes as we do elsewhere in the code.

Reply:
As per our discussion, opened a tracker to handles this separately.